test/services/site_channel_domain_setter_test.rb

```diff
@@ -78,7 +103,13 @@ def teardown
     channel_details = SiteChannelDetails.new
     channel_details.brave_publisher_id_unnormalized = "https://bad url.com"
     SiteChannelDomainSetter.new(channel_details: channel_details).perform
+    assert_equal 'invalid_uri', channel_details.brave_publisher_id_error_code
+  end
 
+  test "raises exception with invalid url without protocol" do
```

Comment:
minor nit - test name indicates raises an exception but sets brave_publisher_id_error_code

The next test name is the same